src/ui/public/vis_maps/geohash_layer.js

```diff
@@ -0,0 +1,574 @@
+import KibanaMapLayer from './kibana_map_layer';
```

Comment:
Clarity, to keep all related code together. These types aren't used anywhere else. imo, they're just implementation details of how we make sure we can display geohash layers.